ios/Podfile.lock

```diff
@@ -419,6 +421,7 @@ DEPENDENCIES:
   - React-RCTVibration (from `../node_modules/react-native/Libraries/Vibration`)
   - ReactCommon/callinvoker (from `../node_modules/react-native/ReactCommon`)
   - ReactCommon/turbomodule/core (from `../node_modules/react-native/ReactCommon`)
+  - "ReactNativeAudioToolkit (from `../node_modules/@react-native-community/audio-toolkit`)"
```

Comment:
Why is this the only one quoted?

Reply:
i haven't edit this maually.. i just did `make shell` -> `yarn add ...` and `make nix-update-pods`
i think all @react-native-community modules has "@" so you have all them quoted

Reply:
I pinged react-native-community lol.

Reply:
I see.